Services/ProductsService.js

```diff
@@ -13,8 +15,13 @@ const getSingleProduct = async (connection, sku) => {
     return response
 }
 
+const updateSingleStockLevel = async (connection, sku, stock_level) => {
+    return await connection.query('UPDATE `products` SET `stock_level` = ' + `'${stock_level}'` + 'WHERE `SKU` = ' + `'${sku}'`)
```

Comment:
If I enter the correct SKU number but then leave the stock level value blank, it crashes the API. Could we add in some code that checks if variables are set and return the error JSON response if not?

Gabriel included the below code in a different function to check if the SKU exists before continuing, it may be helpful:

const singleProduct = await ProductService.getSingleProduct(connection, sku)
const deleteProduct = await ProductService.deleteSingleProduct(connection, sku)
let apiResponse
if (singleProduct.length !== 0 && sku) {
    apiResponse = JsonResponseService(deleteProduct, true, 'Success', 204)
} else {
    apiResponse = JsonResponseService()
}
res.json(apiResponse)

Reply:
True that